spec/swagger_helper.rb

```diff
@@ -92,15 +92,15 @@
                     required: %i[trust_uid school_urns],
                     properties: {
                       trust_uid: {
-                        type: :integer,
-                        example: 12_345,
+                        type: :string,
+                        example: "12345",
                       },
                       school_urns: {
                         type: :array,
                         minItems: 0,
                         items: {
-                          type: :integer,
-                          example: 12_345,
+                          type: :string,
+                          example: "12345",
```

Comment:
isn't URN a number?

Reply:
is it possible that we've just mapped it wrongly in our DB? @scruti I'm not sure, but the data I've seen suggests that these are numbers

Reply:
Looks like the URN and UID we import from the GIAS CSV come as numbers, yes. Was directly introduced like strings 4-5 years ago with the creation of the organisation table.

Not sure if there is any benefit in migrating them to numbers since they're "ids" not used for operations/calculations.